src/process.c

```diff
@@ -392,6 +392,13 @@ pset_stderrproc (struct Lisp_Process *p, Lisp_Object val)
   p->stderrproc = val;
 }
 
+/* Accessors to enable Rust code to get data from the Lisp_Process struct */
+pid_t fget_pid(const struct Lisp_Process *p)
```

Comment:
This should be `pget_pid`. The 'f' in mine is for 'frame'. Look at process.h you can see others.

Reply:
I did not figure it out why the f was there. Thank you! Is fixed now.